RELEASE_NOTES.md

```diff
@@ -2,23 +2,77 @@
 
 While [CHANGELOG.md](./CHANGELOG.md) contains detailed documentation and links to all of the source code changes in a given release, this document is intended to be aimed at a more comprehensible version of the contents of the release from the point of view of users of Refinery.
 
+## Version 2.0.0
+
+This is a major release of Refinery with a new configuration file format, new samplers, and updated runtime behavior. It has several breaking changes, and so it also comes with a conversion tool.
```

Comment:
@TylerHelmuth Would you want to put an internal link to the Conversion Tool section here?

Reply:
Good idea

Reply:
Added